src/flynt/cli.py

```diff
@@ -53,11 +53,12 @@ def main():
     )
 
     parser.add_argument(
-        "-n",
+        "-d",
         "--dry-run",
         action="store_true",
         default=False,
-        help="Do not change the file in-place. Useful when running as a linting process."
+        help="Do not change the file in-place. Note that this must be used in "
```

Comment:
When suggesting this change, I didn't consider we will end up implementing printing diff. Now it's more relevant to state that proposed diff will be printed to stdout, and --dry-run because useful on it's own without --fail-on-change.

Reply:
This should be fixed now.